<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhilipsHueSettingsEntry</class>
 <widget class="QWidget" name="PhilipsHueSettingsEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Philips Hue Settings Entry</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Philips Hue Bridge</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QLabel" name="EntertainmentLabel">
        <property name="text">
         <string>Entertainment Mode:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="EntertainmentCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="AutoConnectLabel">
        <property name="text">
         <string>Auto Connect Group:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QCheckBox" name="AutoConnectCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="IPLabel">
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="ClientKeyValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="ClientKeyLabel">
        <property name="text">
         <string>Client Key:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="UsernameLabel">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="MACLabel">
        <property name="text">
         <string>MAC:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="QPushButton" name="UnpairButton">
        <property name="text">
         <string>Unpair Bridge</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="MACEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="IPEdit"/>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="UsernameValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>IPEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
